e2e/install.e2e.js

```diff
@@ -0,0 +1,49 @@
+const { mkdir } = require('fs').promises
+const { existsSync } = require('fs')
+const { platform } = require('os')
+const { join, resolve } = require('path')
+const process = require('process')
+
+const test = require('ava')
+const execa = require('execa')
+
+const { version } = require('../package.json')
+
+const { packageManagerConfig, packageManagerExists } = require('./utils')
+
+/**
+ * Prepares the workspace for the test suite to run
+ * @param {string} folderName
+ */
+const prepare = async (folderName) => {
+  const folder = join(process.env.E2E_TEST_WORKSPACE, folderName)
+  await mkdir(folder, { recursive: true })
+  return folder
+}
+
+Object.entries(packageManagerConfig).forEach(([packageManager, { install: installCmd, lockFile }]) => {
+  /** @type {import('ava').TestInterface} */
+  const testSuite = packageManagerExists(packageManager) ? test : test.skip
```

Comment:
test will be skipped if package manager is not installed (useful if someone has not installed pnpm for example locally)